lib/eclipse/share/keywords/000_Eclipse100/F/FRICTION

```diff
@@ -0,0 +1,4 @@
+     {"name" : "FRICTION", "sections" : ["RUNSPEC"], "size" : 1, "items" : [
+         {"name" : "NWFRIC"       , "value_type" : "INT" , "default" : 0},
```

Comment:
The names are given in the definition, while not totally sure how they are used or interact with other runspec options. It will be okay to me if it is suggested to change to more meaningful names here.

Reply:
> The names are given in the definition,

That is what we usually use - this looks fine.